Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bevy_enum_filter"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
I believe the version should be `0.3.0` since upgrading Bevy is a breaking change.

Reply:
Okay, absolutely!
Like the previous merge I wasn't sure whether to do this or not.
I updated the README too so will change that accordingly.

Reply:
Yeah normally I prefer to update the version numbers myself, but I've been super busy IRL and so I don't trust myself to even remember to do something as simple as that lol

Reply:
I've updated the version number to `0.3.0` as requested :)